### ISSUE TITLE
Support multiple realsense cameras

### ISSUE THREAD

**Issue by yambati03** [RoboJackets/urc-software#215]
There's already an open PR for this, just need to clean it up and verify its functionality.

**Comment by yambati03** [comment]
@mrinalTheCoder Do we still have multiple realsenses on the rover?

**Comment by mrinalTheCoder** [comment]
> [@mrinalTheCoder](https://github.com/mrinalTheCoder) Do we still have multiple realsenses on the rover?

I don't think so, we can delete this issue. We only need one Realsense for computer vision and ARUCO tag detection.
